jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -441,60 +441,68 @@ private InterpreterResult executeSql(String propertyKey, String sql,
     String paragraphId = interpreterContext.getParagraphId();
     String user = interpreterContext.getAuthenticationInfo().getUser();
 
+    InterpreterResult interpreterResult = new InterpreterResult(InterpreterResult.Code.SUCCESS);
+
     try {
-      String results = null;
       connection = getConnection(propertyKey, interpreterContext);
-
       if (connection == null) {
         return new InterpreterResult(Code.ERROR, "Prefix not found.");
       }
 
-      statement = connection.createStatement();
-      if (statement == null) {
-        return new InterpreterResult(Code.ERROR, "Prefix not found.");
-      }
+      String[] multipleSqlArray = sql.split("\\s*;\\s*(?=([^'\"`]*'[^'\"`]*')*[^'\"`]*$)");
```

Comment:
I just doubt whether this regular expression can cover all the cases.  It would be better to have unit test for this.